guides/install.md

```diff
@@ -26,4 +26,22 @@ With SideStore downloader installed (and it's requirements met), simply connect
 You must then enable "Developer Mode" in your iOS security settings. Additionally, you must *Trust* your personal developer certificate.
 
 Finally, open the SideStore app on your homescreen, re-enter the Apple ID credentials used previously, and refresh to ensure that everything is working correctly.
--->
\ No newline at end of file
+-->
+
+# Pairing Process
+
+Using Jitterbugpair (not jitterbug app) allows us to create a pairing file for programs, like sidestore to talk to your device and is required to use sidestore or it will not function. (Windows and MacOS requires iTunes)
```

Comment:
Doesn't mac come with iTunes/Music reinstalled? And I think jitterbugpair just uses libimobiledevice. So is it even required on windows?

Reply:
> Doesn't mac come with iTunes/Music preinstalled? And I think jitterbugpair just uses libimobiledevice. So is iTunes even required on windows?

Yes it’s required on windows. I’m just listing what jitterbugpair requires on their GitHub. I don’t have a mac to know that part and if older mac versions don’t need it.

Reply:
Does latest changes fixes your problem?